src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocsummarylocation/InputJavadocSummaryLocationTrailingSpace.java

```diff
@@ -0,0 +1,25 @@
+package com.puppycrawl.tools.checkstyle.checks.javadoc.javadocsummarylocation;
+
+/**
+ * config: default
+ */
+public final class InputJavadocSummaryLocationTrailingSpace {
+
+    /**  
+     * ^^ There is a trailing space in the first line. // OK
+     */
+    private int space;
+
+    /**********************
+     * Trailing asterisks in the first line * // OK
+     *
+     **********************/
+    private int asterisks;
+
+    /*** *
+     * There is a space in the first line, weird corner case, we treat it as // violation
```

Comment:
`/*** *` => `/** `

if you want to have case with extra `*`, please make it separate, but first line should be in sync with comment on second line

Reply:
done. Split into two cases.